docker/pre-install

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Modified version of github.com/spdk/docker
+
+set -e
+
+spdk_repo=/tmp/spdk
+
+cleanup() {
+
+	rm -f "$HOME/rpmbuild/rpm/x86_64/"*.rpm
+	rm -f "$spdk_tar"
+}
+
+trap 'cleanup' EXIT
+
+# Spice it a bit with supported sources
+"$spdk_repo/scripts/pkgdep.sh" -d
```

Comment:
This does not install librbd dependencies. Some time ago I created a pull request to add `--rbd` option for this [1] but it has not been merged.

[1]  https://review.spdk.io/gerrit/c/spdk/spdk/+/11134